jdbc-http/src/main/java/com/salesforce/datacloud/jdbc/auth/DataCloudTokenProvider.java

```diff
@@ -136,8 +141,8 @@ private FormCommand buildAuthenticate() throws SQLException {
                 builder.bodyEntry("password", settings.getPassword());
                 break;
             case PRIVATE_KEY:
-                // https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_jwt_flow.htm
-                builder.bodyEntry("grant_type", "urn:salesforce:grant-type:external:cdp");
+                // https://help.salesforce.com/s/articleView?id=xcloud.remoteaccess_oauth_jwt_flow.htm
+                builder.bodyEntry("grant_type", "urn:ietf:params:oauth:grant-type:jwt-bearer");
```

Comment:
unrelated change? Why? This is not described in the commit message at all

Reply:
In the [spec](https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_jwt_flow.htm), to get the access token, we need to use grant_type as `urn:ietf:params:oauth:grant-type:jwt-bearer`.